build.sbt

```diff
@@ -1,55 +1,77 @@
 // basics
 
-name := "scala-logging"
-crossScalaVersions := Seq("3.3.1", "2.11.12", "2.12.18", "2.13.12")
-scalaVersion := crossScalaVersions.value.head
-ThisBuild / versionScheme := Some("early-semver")
-scalacOptions ++= Seq(
-  "-unchecked",
-  "-deprecation",
-  "-language:_",
-  "-encoding", "UTF-8",
-  "-Ywarn-unused"
-)
-incOptions := incOptions.value.withLogRecompileOnMacro(false)
+val scala213 = "2.13.12"
+val scala3 = "3.4.0-RC1-bin-20231016-69e1338-NIGHTLY" //TODO 3.3.2 or 3.4.0
```

Comment:
once a release includes https://github.com/lampepfl/dotty/pull/18663 we can bump this, retest and release